cli/tools/test_runner.rs

```diff
@@ -304,37 +303,30 @@ pub async fn run_test_file(
   Ok(())
 }
 
+/// Runs tests.
+///
+/// Returns a boolean indicating whether the tests failed.
```

Comment:
whether the tests ran successfully (the current wording makes it seem like true = failed)

Reply:
`true` does mean failed, mostly just because that was the value which was already there.

Reply:
It's returning `has_failure`.
Good enough to land imho, I'll be refactoring this soon anyway so wouldn't let it block.